README.md

```diff
@@ -66,16 +66,18 @@ If you’d prefer to compile from source code please follow [these instructions]
 
 ## Compatibility
 
-Full Lightning Terminal functionality can be dependent on running a certain
-version of `lnd`. View the table below to ensure that you run the correct
+Full Lightning Terminal functionality can be dependent on running a compatible
+version of `lnd`. View the table below to ensure that you run a compatible
 version of `lnd` with the relevant `litd` release.
 
-This version compatibility dependency is only relevant for running remote mode
-`litd`. The bundled version will always come with the correct, [compatible
-versioning](#daemon-versions-packaged-with-lit).
+This version compatibility dependency table is only relevant when running `lnd`
+in remote mode, and shows the **_minimum_** `lnd` version that can be used with
```

Comment:
"when running `lnd` in remote mode" -> technically LND cant be run in remote mode. It is a mode of "litd". So perhaps just "when running `litd` in remote mode" or if we want to be more descriptive: "when running `litd` in remote mode (meaning that `lnd` is running as a separate process to `litd`)?

Reply:
Thanks for the good clarification suggestion :)! I updated this, but changed the "(meaning that ... )" section a little to:
"This version compatibility dependency table is only relevant when running litd in remote mode (meaning that `lnd-mode=remote` is set)."

I think that should be pretty clear to most users. The only con with my new version is that we don't mention the operating modes of Litd until after the table, but I think it should be understandable either way.

If we would like to address that con though, I think it would also make sense to move the entire section of "Full Lightning Terminal functionality can be dependent on running a compatible version of `lnd` ..." + the version compatibility dependency table. I'd suggest we'd then move it so it's written after the "supported combinations table".

Let me know if you think the current version is ok, or if you'd like me to move the section!